holidays/countries/kazakhstan.py

```diff
@@ -195,6 +196,7 @@ class KazakhstanIslamicHolidays(_CustomIslamicHolidays):
         2022: (JUL, 9),
         2023: (JUN, 28),
         2024: (JUN, 16),
+        2025: (JUN, 6),
```

Comment:
You might still want to add the source for Kazakhstan's 2025 Kurban Ait/Eid al-Adha date though i.e. https://en.tengrinews.kz/kazakhstan_news/ramadan-2025-start-date-announced-for-kazakhstan-264556/